CHANGELOG.md

```diff
@@ -23,7 +23,7 @@ Main (unreleased)
 - `otelcol.exporter.prometheus`: Set `include_scope_info` to `false` by default. You can set 
   it to `true` to preserve previous behavior. (@gouthamve)
 
-- `prometheus.remote_write`: Set `retry_on_http_429` to `true` by default in the `queue_config` block.
+- `prometheus.remote_write`: Set `retry_on_http_429` to `true` by default in the `queue_config` block for flow and static modes.
```

Comment:
There is no `prometheus.remote_write` in static mode, so I think this should be rephrased. 
Also in the release notes page we should make it more clear that it's flow and static mode: https://grafana.com/docs/agent/next/flow/release-notes/#breaking-change-the-default-value-of-retry_on_http_429-is-changed-to-true-for-the-queue_config-in-prometheusremote_write

Reply:
I updated it and for the static mode I added it in the release notes of static mode. Is it what you had in mind?